CONTRIBUTING.md

```diff
@@ -21,7 +21,7 @@ For Git commit messages, our principle is that `git log --pretty=oneline` should
   * *Improve explanation of …*
   * *Remove module X because it is not used.*
 * Commits should have a description that concisely explains the rationale and context for the change if that is not obvious.
-* The first line of the description should include a reference to the fixed issue `(#XXX)`.
+* Commit descriptions should include a `Fixes #XX` line indicating what GitHub issue number the commit fixes.
```

Comment:
```suggestion
* Commit descriptions can include a `Fixes #XX` line indicating what GitHub issue number the commit fixes.
```

Reply:
As discussed with Samir this was a revert of CONTRIBUTING.md.